### ISSUE TITLE
rollover with Touch Punch without moving shapeshift element

### ISSUE THREAD

**Issue by nliapis** [AshesOfOwls/jquery.shapeshift#62]
Hello again i have use jQuery Touch Punch for touch support for ipad. i would like to ask if i can  rollover in ipad without moving an shapeshift element. I mean to rollover over an shapeshift element without move it.

Thanks and the plugin is super.

**Comment by AshesOfOwls** [comment]
Unfortunately thats just a limitation of touch devices not being able to have a hover option. Sometimes a quick tap can activate the hover state but other than that I can't think of any work arounds, sorry.

Thanks for the support!

**Comment by jackmarketon** [comment]
A touch device could do a tap vs double tap, which could be implimented but would require something like modernizr to test for it...

**Comment by AshesOfOwls** [comment]
Yeah that could work, it could also detect how long the person touches the touches the element and only begin the movement when its past a certain time, otherwise show the hover. It's possible to include this in the 3.0 todo but I am not 100% confident about finishing it.

**Comment by nliapis** [comment]
Thank you guys, McPants when you will release 3.0?

**Comment by AshesOfOwls** [comment]
The goal was the end of this month but I am trying not to release it prematurely. If I had to guess it would be before the end of August.
